openapi.json

```diff
@@ -7,7 +7,7 @@
             "name": "MIT License",
             "url": "https://mit-license.org/"
         },
-        "version": "0.0.post1.dev792+g76f08b5.d20230417"
+        "version": "unknown"
```

Comment:
Hmm why is the version now unknown?

Reply:
It's most likely coming from [here](https://github.com/DataJunction/dj/blob/main/dj/__init__.py#L12). Could it be that some of the package manager changes that were recently made is making this logic hit the `PackageNotFoundError` now? I'll look into it--I think we ideally want the generated API spec to be mapped to the major version of the library.

Reply:
Oh the comment there actually describes exactly what's happening, lol
```py
# Change here if project is renamed and does not equal the package name
```
This is probably when we changed the package name to `datajunction-server` since `dj` was taken in PyPI (which was grandfathered in I think since they don't allow two character package names anymore). It always feels worth it to ensure the import name always matches the pypi name. Any objections to me changing `from dj import ...` to `from datajunction_server import ...` across the code-base?

Reply:
No objection, although I think this was my wishful thinking. You gotta do what you gotta do.